backend/LexData/SeedingData.cs

```diff
@@ -146,13 +147,15 @@ private async Task SeedUserData(CancellationToken cancellationToken = default)
             LastCommit = DateTimeOffset.UtcNow,
             RetentionPolicy = RetentionPolicy.Dev,
             IsConfidential = false,
+            Organizations = [],
             Users = [],
         });
 
         lexBoxDbContext.Attach(new Organization
         {
             Id = new Guid("292c80e6-a815-4cd1-9ea2-34bd01274de6"),
             Name = "Test Org",
+            Projects = [],
```

Comment:
I'd like to have one of our test projects be part of an org on seeding.

Reply:
I've got a SeedingData change that I haven't committed yet creating a second organization; I'm going to have one org own sena-3 and the other org own nothing, and have elawa not owned by an org. That way all the combinations will be covered — and if we need more combinations, I'll create a third test project.

Reply:
Second test org created in commit 451db73e, and sena-3 assigned to test org in commit fea02c05.